src/actions/CommandAction.ts

```diff
@@ -69,7 +73,7 @@ export default class CommandAction implements Action {
 
   async run(
     [targets]: [TypedSelection[]],
-    { showDecorations = true } = {}
+    { command = "", showDecorations = true } = {}
```

Comment:
As above, I'd probably just make this one optional or nullable and then use `??` below